### PR TITLE
Fix GOVUK Rubocop RSpec/ExampleWording Offences

### PR DESCRIPTION
**Pull request by chris-groves**
Fix GOVUK Rubocop RSpec/ExampleWording Offences.

RSpec/ExampleWording: Do not use should when describing your tests. (https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/ExampleWording)

RSpec/ExampleWording: Do not repeat 'it' when describing your tests. (https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/ExampleWording)

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing: `bundle exec rake`
- Github should not be reporting conflicts; you should have recently run `git rebase main`.
- There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- The PR description should say what you changed and why, with a link to the JIRA story.
- You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: spec/forms/applicants/basic_details_form_spec.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
   subject { described_class.new(params) }
 
   describe ".model_name" do
-    it 'should be "Applicant"' do
+    it 'is an "Applicant"' do
```
Comment (colinbruce) [inline spec/forms/applicants/basic_details_form_spec.rb:7]: <pedantic>Shouldn't this be 'Is "Applicant"'</pedantic>
We are checking the contents of a string, not the object type?

File: spec/forms/applicants/email_form_spec.rb [hunk 2]
```diff
@@ -4,7 +4,7 @@
   subject { described_class.new(params) }
 
   describe ".model_name" do
-    it 'should be "Applicant"' do
+    it 'is an "Applicant"' do
```
Comment (colinbruce) [inline spec/forms/applicants/email_form_spec.rb:7]: Ditto above

File: spec/forms/legal_aid_applications/own_home_form_spec.rb [hunk 3]
```diff
@@ -9,7 +9,7 @@
   let(:form_params) { params.merge(model: application) }
 
   describe ".model_name" do
-    it 'should be "LegalAidApplication"' do
+    it 'is a "LegalAidApplication"' do
```
Comment (colinbruce) [inline spec/forms/legal_aid_applications/own_home_form_spec.rb:12]: Ditto above

File: spec/forms/legal_aid_applications/shared_ownership_form_spec.rb [hunk 4]
```diff
@@ -11,7 +11,7 @@
   end
 
   describe ".model_name" do
-    it 'should be "LegalAidApplication"' do
+    it 'is a "LegalAidApplication"' do
```
Comment (colinbruce) [inline spec/forms/legal_aid_applications/shared_ownership_form_spec.rb:14]: Ditto above

**Review by colinbruce — APPROVED** [review]
One pedantic observation about a couple of the test names - non-blocking
Otherwise, LGTM :+1:

**Review by kmahern — APPROVED** [review]
👍

**Comment by chris-groves** [comment]
@colinbruce I've made the suggested changes

**Pull request merged**